sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -83,6 +83,7 @@ def __init__(
         if service not in ["blob", "queue", "file-share", "dfs"]:
             raise ValueError("Invalid service: {}".format(service))
         service_name = service.split('-')[0]
+        # account = parsed_url.netloc.split(".{}.preprod.core.".format(service_name))
```

Comment:
Are we saving this for something? TODO if that's the case, delete if not.

Reply:
I'll remove that.